python/TestHarness/TestHarness.py

```diff
@@ -444,240 +339,113 @@ def checkExpectError(self, output, expect_error):
         else:
             return True
 
-    # PBS Defs
-    def checkPBSEmulator(self):
-        try:
-            qstat_process = subprocess.Popen(['qstat', '--version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            qstat_output = qstat_process.communicate()
-        except OSError:
-            # qstat binary is not available
-            print('qstat not available. Perhaps you need to load the PBS module?')
-            sys.exit(1)
-        if len(qstat_output[1]):
-            # The PBS Emulator has no --version argument, and thus returns output to stderr
-            return True
+    # handleTestStatus is the entry point the testers will use to print statuses to the screen
+    def handleTestStatus(self, tester):
+        # The test has finished regardless of status
+        if tester.isFinished():
+            if tester.isSilent() or (tester.isDeleted() and not self.options.extra_info):
+                return
+            else:
+                self.testOutputAndFinish(tester)
+
+        # The test has not yet finished but there is a status change we want to display
         else:
-            return False
+            print printResult(tester, tester.getStatusMessage(), clock() - tester.getStartTime(), self.options)
```

Comment:
`printResult` doesn't actually print anything. This should also use new `print()`